### PR TITLE
STDTC-29: add tests for Progress with jest/react-testing-library

### PR DESCRIPTION
**Pull request by UladzislauKutarkin**
## Purpose

https://issues.folio.org/browse/STDTC-29
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Review by vashjs — APPROVED** [review]

File: lib/Progress/Progress.js [hunk 1]
```diff
@@ -41,14 +41,20 @@ export const Progress = memo(({
...
           className={progressCurrentClassName}
           style={{ width: `${progressValue}%` }}
+          data-testid="progress-current"
         />
       </div>
-      <div className={progressInfoClassName}>{progressInfo}</div>
+      <div
+        data-testid="progress-info"
+        className={progressInfoClassName}
+      >{progressInfo}
```
Comment (NikitaSedyx) [inline lib/Progress/Progress.js:56]: bad codestyle
<div>
  {info}
</div>

File: lib/Progress/Progress.test.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    renderDefaultProgress();
+
+    expect(screen.getByText(`${totalValue / currentValue}%`)).toBeVisible();
+  });
+
+  describe('rendering Progress with "messagedPercentage" progress info type', () => {
+    it('should display correct progress info', () => {
+      const testMessage = 'TestMessage';
+
+      render(<Progress
```
Comment (NikitaSedyx) [inline lib/Progress/Progress.test.js:44]: looks like a copypaste of renderDefaultProgress, change it to renderProgress and support props with default values

File: lib/Progress/Progress.js (hunk 1 above)
Comment (UladzislauKutarkin) [reply lib/Progress/Progress.js:56]: You mean data-testid ?

Comment (UladzislauKutarkin) [reply lib/Progress/Progress.js:56]: > bad codestyle
> 
> {info}

You mean data-testid ?

Comment (UladzislauKutarkin) [reply lib/Progress/Progress.js]: done.

File: lib/Progress/Progress.test.js (hunk 2 above)
Comment (UladzislauKutarkin) [reply lib/Progress/Progress.test.js]: done.

> looks like a copypaste of renderDefaultProgress, change it to renderProgress and support props with default values

done.

**Review by NikitaSedyx — APPROVED** [review]

**Pull request merged**